querybook/webapp/lib/utils/embed.ts

```diff
@@ -0,0 +1,4 @@
+const EMBED_PATH_STRING = 'embedded';
```

Comment:
this is a bit too relaxed? since you could have datadoc that has embedded in the title and that would modify the url to have embedded as string. It is better to check if the path contains `/_/embedded`

Reply:
Good point, updated